cypress/common/base.ts

```diff
@@ -621,6 +621,19 @@ export class BaseMethods {
         })
     }
 
+    public universalCheckMethod({
```

Comment:
not sure about this idea. Too vague, with this method we could delete a lot methods from base.ts but I think it would not be a best practice

Reply:
Fixed